make.js

```diff
@@ -345,8 +345,8 @@ target.testLegacy = function() {
     run('mocha ' + testsSpecPath, /*inheritStreams:*/true);
 }
 
-target.package = function() {
-    // validate powershell 5
+target.layout = function() {
```

Comment:
Thanks for the suggesting to make use of --version. Will update the PR making sure that if version is not specified then just log it to the screen and exit the process.